src/main/java/apoc/date/Date.java

```diff
@@ -136,6 +136,19 @@ public String systemTimezone() {
 		return TimeZone.getDefault().getID();
 	}
 
+	@UserFunction
+	@Description("apoc.date.convert(12345,'ms|s|m|h|d','ms|s|m|h|d') convert a time unit into a different time unit")
+	public Long convert(@Name("time") long time, @Name(value = "unit") String unit, @Name(value = "toUnit") String toUnit) {
```

Comment:
Maybe 'ms' for the `unit` value, but I'm wondering if in this case a default value would be easier to work with or make usage more error-prone.